rcongui/src/components/UserSettings/autoMods.js

```diff
@@ -362,7 +366,7 @@ export const SeedingAutoMod = ({
         {player_name}, {received_warnings}, {max_warnings}, {next_check_seconds} and {violation}
         are the allowed message parameters
     */
-    "warning_message": "Warning, {player_name}! You violate seeding rules on this server: {violation}\nYou will be punished after {max_warnings} warnings (you already received {received_warnings}), then kicked.\nNext check will happen automatically in {next_check_seconds}s.",
+    "warning_message": "Warning, {player_name}! You violate seeding rules on this server: {violation}\\nYou will be punished after {max_warnings} warnings (you already received {received_warnings}), then kicked.\\nNext check will happen automatically in {next_check_seconds}s.",
```

Comment:
Not sure about adding extra "\" (?)

Reply:
> Not sure about adding these extra backslashes (?)

If you go to `https://localhost:9010/#/settings/automod-seeding` and scroll to see the Documentation, you'll see that part of the messages will go on a new line because "`\n`" gets interpreted as a new line, not as part of the string.
In all the other instances, "`\n`" is escaped as well with "`\\n`".